pre-commit

```diff
@@ -22,6 +24,16 @@ module PreCommit
     end
   end
 
+  def generate_color_docs
+    changed_files.each do |path|
+      if COLOR_DOC_SOURCE_FILE_PATHS.include? path
+        puts "Color system changes detected. Generating new docs..."
```

Comment:
It could be useful to mention this may take some time, to manage the user's expectations.

Reply:
```suggestion
        puts "Color system changes detected. Generating new docs (may take a couple minutes)..."
```